Firestore/core/src/firebase/firestore/remote/serializer.h

```diff
@@ -72,7 +72,7 @@ class Serializer {
   // TODO(rsgowman): If we never support any output except to a vector, it may
   // make sense to have Serializer own the vector and provide an accessor rather
   // than asking the user to create it first.
-  static util::Status EncodeFieldValue(
```

Comment:
could you elaborate why remove `static` here?

Reply:
The tests instantiate the serializer and then access these static methods through the instance, which causes a warning under clang-tidy.

In the fullness of time the serializer will need to be stateful (for example, to encode a DocumentKey you need to know the current DatabaseId) so fixing those warnings by removing the static seemed like the better way.